azext_iot/operations/hub.py

```diff
@@ -3301,3 +3303,311 @@ def _customize_device_tracing_output(device_id, desired, reported):
         ):
             output["isSynced"] = True
     return output
+
+
+# Topic Space Preview
+
+
+def iot_hub_topic_space_create(
+    cmd,
+    topic_name,
+    topic_templates,
+    topic_type,
+    hub_name=None,
+    resource_group_name=None,
+    login=None,
+    auth_type_dataplane=None,
+    target=None,
```

Comment:
What is the purpose of target here? Why is it exposed to users?

Reply:
Same thing for service sdk?

Reply:
Was going to reuse the code here for update but will remove this.